### ISSUE TITLE
Desktop Layout

### ISSUE THREAD

**Issue by stdavis** [agrc/recreate-web#34]
The app currently looks good only on smaller devices. We need a larger screen design that includes showing the list and map at the same time.

**Comment by brigsz** [comment]
So you feel this needs to be done for an ipad or small tablet?

**Comment by stdavis** [comment]
Yes, but I also think that it should look decent on a desktop size screen.

**Comment by stdavis** [comment]
Desktop size is no long applicable (#35).
